<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGeneratorDialogue</class>
 <widget class="QDialog" name="PasswordGeneratorDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="vLayout_Generator">
     <item>
      <widget class="QLineEdit" name="lineEdit_Password"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Generate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Menlo</family>
        </font>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CopyPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Menlo</family>
        </font>
       </property>
       <property name="text">
        <string>Copy Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Length">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Menlo</family>
        </font>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hSlider_Length">
       <property name="maximum">
        <number>4086</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Seed">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Menlo</family>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Seed</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout_Alphabetical">
       <item>
        <widget class="QGroupBox" name="groupBox_AZ">
         <property name="font">
          <font>
           <family>Menlo</family>
          </font>
         </property>
         <property name="title">
          <string>A-Z</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QSpinBox" name="spinBox_AZ">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Menlo</family>
             </font>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_AZ">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_az">
         <property name="font">
          <font>
           <family>Menlo</family>
          </font>
         </property>
         <property name="title">
          <string>a-z</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBox_az">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Menlo</family>
             </font>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_az">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout_Specials">
       <item>
        <widget class="QGroupBox" name="groupBox_Special2">
         <property name="font">
          <font>
           <family>Menlo</family>
          </font>
         </property>
         <property name="title">
          <string> Special2</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSpinBox" name="spinBox_Special2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Menlo</family>
             </font>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_Special2">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Special">
         <property name="font">
          <font>
           <family>Menlo</family>
          </font>
         </property>
         <property name="title">
          <string>Special</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBox_Special">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Menlo</family>
             </font>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_Special">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_09">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Menlo</family>
        </font>
       </property>
       <property name="title">
        <string>0-9</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="spinBox_09">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Menlo</family>
           </font>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hSlider_09">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
